scheduler/scheduler-node/src/test/java/functionaltests/ForkedTaskExecutorTest.java

```diff
@@ -324,6 +326,17 @@ private Decrypter createCredentials(String username) throws NoSuchAlgorithmExcep
 
     @Before
     public void setUp() throws Exception {
+
```

Comment:
normally all tests are started in gradle with the correct property paths set. Did you have an issue while running the test in intellij ? If yes, then you simply need to add these properties as test jvm parameters rather than change the code.